install/install.sh

```diff
@@ -25,14 +25,16 @@ if [ -n "${S3_BUILD_CACHE_KEY}" ]; then
 fi
 spack mirror list
 
-# Add flux, pytest and flake8 to chiltepin spack environment
+# Add flux, pytest, flake8, and black to chiltepin spack environment
 spack add python
 spack add py-pip
 spack add py-pytest
 spack add flux-core@0.53.0
 spack add flux-sched@0.28.0
 spack add py-flake8
 spack add py-pytest-flake8
+spack add py-black
+spack add py-pytest-black
```

Comment:
I wasn't able to find the exact command for adding `black`. I know you previously found the command for `flake8` and was hoping you could point me in the right direction. For now, I assumed it would look almost identical to the rest of the commands. 

https://spack.readthedocs.io/en/latest/search.html?q=black&check_keywords=yes&area=default